MAUI/DataForm/Editing.md

```diff
@@ -0,0 +1,452 @@
+---
+layout: post
+title: Commit in .NET MAUI DataForm control | Syncfusion
+description: Learn here all about the commit feature in Syncfusion .NET MAUI DataForm (SfDataForm) control and more.
+platform: maui
+control: SfDataForm
+documentation: ug
+---
+
+# Editing in .NET MAUI DataForm (SfDataForm)
+
+The data form supports several built-in editors.
+
+## Supported editors and associated DataFormItem
```

Comment:
Remove this
## Supported editors and associated DataFormItem

Reply:
Removed